docs/api/api-spec.md

```diff
@@ -142,18 +142,41 @@ Context channels allows a set of apps to share a stateful piece of data between
 
-1. The 'system' ones, which have a well understood identity. One is called 'default'.
+1. The 'system' ones, which have a well understood identity. One is called 'global'.
 2. The 'app' ones, which have a transient identity and need to be revealed
-
-The 'system' channels include a 'default' channel which serves as the backwards compatible layer with the 'send/broadcast context' above. There are some reserved channel names for future use. Currently this is just 'global'.
+ 
 
 ### Joining Channels
-Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel, except if the channel joined is the 'default'.
+Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel.
 
-When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  It is up to the desktop agent to determine the behavior of the 'default' channel, and if context is automaticaly broadcast over it or not.
+When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not joined to a channel these methods will be no-ops, but apps can still choose to listen and broadcast to specific channels via the methods on the `Channel` class.  
 
 It is possible that a call to join a channel could be rejected.  If for example, the desktop agent wanted to implement controls around what data apps can access.  
 
+Joining channels in FDC3 is intended to be a behavior initiated by the end user. For example: by color linking or apps being grouped in the same workspace.  Most of the time, it is expected that apps will be joined to a channel by mechanisms outside of the app.  Always, there SHOULD be a clear UX indicator of what channel an app is joined to. 
+
+### The 'global' Channel
+The 'system' channels include a 'global' channel which serves as the backwards compatible layer with the 'send/broadcast context' behavior in FDC3 1.0.  A desktop agent MAY choose to make membership in the 'global' channel the default state for apps on start up.  
+
+The 'global' channel should be returned as part of the response from the `fdc3.getSystemChannels` call.  Desktop Agents may want to filter out the 'global' option in their UI for system channel pickers.
+
+
+#### Examples
+
+An app gets the current context of the `global` as a fallback if no other context is set.
+
+```js
+const globalChannel = await fdc3.getOrCreateChannel("global");
+const context = await globalChannel.getCurrentContext("fdc3.instrument");
+```
+
+An app wants to respond to all context changes, whether on a joined channel or the `global` channel.
```

Comment:
I also don't understand what is sentence mean and what feature/behavior the below example demonstrates. Can you just expand a bit?

Reply:
@vmehtafds . This example is showing an app listening to the global channel *outside* of whatever channel it may be joined to.  This example is meant to illustrate context scopes - i.e. I may have a *joined channel scope* and a *global scope*.   For example, in a multi-windowed application, I may have a top-level search bar that represents global scope and individual windows may be on their own individual context  scopes, or joined to a channel-wide context scope.  This may or may not be useful for you.  I am happy to get other examples in addition to or instead of this one.

Reply:
I agree with @vmehtafds here: `An app wants to respond to all context changes, whether on a joined channel or the `global` channel.`

This is very confusing to me. 
- If you are joined to the global channel, you will get its context changes. 
- If you are joined to the red channel you will get its context changes. 
- If you are joined to no channels you will get no context changes.

I think this idea of scoping etc. is unnecessary complexity at this stage and I don't believe it was something originally in the spec, or proposed before. We should just make channels mutually exclusive.

If you do want this type of behaviour (which I think is unnecessary), it should be done by passing an array of channel names to join, and further API support for figuring out how channels are scoped/which "set" of channels you are in.

Reply:
@rikoe @vmehtafds   
"Channel scoping" is not something proposing for the spec - its something that someone may (or may not) implement on top of it.  This is meant to be an *implementation* example.  

This example seems to be confusing and taking us down a rabbit hole.  I'm happy to get other suggestions of how to illustrate the API.

Reply:
Changed `An app wants to respond to all context changes, whether on a joined channel or the `global` channel.` to: `An app wants to explicitly receive context events on the global channel, regardless of what channel it is currently joined to (if any).` (Or something like that. Apart from that the example is fine.)

Reply:
@rikoe  @vmehtafds  - just pushed some clarifications to the language on this example.   Please keep the feedback coming!